docs/docs/docs/01-core/admin/01-ee/02-hooks/use-license-limits.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: useLicenseLimits
+description: API reference for the useLicenseLimits hook
+tags:
+  - admin
+  - hooks
+  - users
+  - ee
+---
+
+An abstraction around `react-query`'s `useQuery` hook to fetch license limits for a project.
+
+## Usage
+
+```js
+const { license, isError, isLoading } = useLicenseLimits();
+```
+
+### `license`
+
+An object the contains the whole admin API response.
```

Comment:
Can we type the object? even if it's a first draft / rough example?

Reply:
Done & rebased.